charts/core/src/Tooltip/Tooltip.tsx

```diff
@@ -35,6 +35,7 @@ export function Tooltip({
         borderRadius: borderRadius[200],
         borderWidth: 0,
         confine: true,
+        appendTo: 'body',
```

Comment:
I'm not sure I'm fully following. I don't think it would cause the container to increase or decrease, so I don't believe it would do that to the page. That said, we can provide any selector we choose, so I think if we give `ChartCard` an ID, it's totally fine to instead append the tooltip to the card. I assume if the selector doesn't exist it will just gracefully fall back to the default. (Did test this locally and it appears to work just fine)

Reply:
Going to merge this in, but we can continue chatting and if there is an issue I'd be happy to push up a patch!